doc/man7/password-handling.pod

```diff
@@ -0,0 +1,130 @@
+=pod
+
+=head1 NAME
+
+password handling - How diverse parts of OpenSSL treat pass phrases
```

Comment:
Maybe replace "password handling" -> "password encoding", and password-handling.pod -> password-encoding.pod? The NAME section would become

"password encoding - How diverse parts of OpenSSL encode pass phrases"

Reply:
_Every_ string is encoded, i.e.  its characters are mapped to bit sequences. What you meant to say is that OpenSSL does no re-encoding.  But never mind, it was just an idea.

Reply:
Right.  You're construct has "diverse parts of OpenSSL" the subject and "encode" the verb, at least the way I read it.  However, I got to think about this another round, and have hopefully come up with something that will satisfy us both.